docs/tutorial/tutorial-0.rst

```diff
@@ -10,6 +10,7 @@ Set up your development environment
 ===================================
 
 Open a command prompt on your computer and make sure that you can successfully run the :code:`python3` command. Create a working directory for your code and change to it.
+If Python 3 is *not* installed, you can do so via `the official installer <https://www.python.org/downloads/>`_, or via `pyenv <https://github.com/pyenv/pyenv#simple-python-version-management-pyenv/>`_, as described in the `environment page <https://pybee.org/contributing/how/first-time/setup/>`_.
```

Comment:
The link to pyenv doesn't work properly. I think you need to either remove the trailing / or link to the main page (instead of the section on the page).

Reply:
Should be fixed now